### DISCUSSION TITLE
Unable to view new tenants added

### DISCUSSION THREAD

**Discussion by JKFISH1** [KelvinTegelaar/CIPP#1681] [Q&A]
We have added 2 new tenants to our partner center, unfortunately, they're not visible in CIPP.

Steps taken:
Waited 24+ hours
Tenant Cache cleared
Browser Cache cleared
Different browser used
Added Azure plan to one of the tenants

**Comment by KelvinTegelaar** [comment]
Support is only available for sponsors
